src/app/sign-in/pages/sign-in/sign-in.component.ts

```diff
@@ -52,6 +53,7 @@ export class SignInComponent implements OnInit {
   ) {}
 
   ngOnInit() {
+    this.formSignInComponent.onSubmit()
```

Comment:
is this correct @leomendoza123? Do we wanna to call the onsubmit method in the ngOnInit